components/clp-package-utils/clp_package_utils/general.py

```diff
@@ -94,6 +94,7 @@ def __init__(self, clp_home: pathlib.Path, docker_clp_home: pathlib.Path):
         self.archives_output_dir: typing.Optional[DockerMount] = None
         self.stream_output_dir: typing.Optional[DockerMount] = None
         self.aws_config_dir: typing.Optional[DockerMount] = None
+        self.generated_config_dir: typing.Optional[DockerMount] = None
```

Comment:
I guess first, this should be `generated_config_file` instead of dir.

And I guess it might be better to just mount the parent of generated_config_file? (so mount a dir).
In our current code, it won't do anything since generated_config_file.parent would be the same as logs_dir, but it can prevent some silly future error.